vendors/swedbank-pay-core-library/src/SwedbankPay/Core/Data.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+namespace SwedbankPay\Core;
+
+use InvalidArgumentException;
+
+class Data implements \ArrayAccess
+{
+    protected $data = [];
+
+    /**
+     * Check is data exists
+     * @param $key
+     * @return bool
+     */
+    public function hasData($key)
+    {
+        return isset($this->data[$key]);
+    }
+
+    /**
+     * Get Data
+     * @param mixed $key
+     * @return array|mixed
+     */
+    public function getData($key = null)
+    {
+        if (!$key) {
+            return $this->data;
+        }
+
+        return isset($this->data[$key]) ? $this->data[$key] : null;
+    }
+
+    /**
+     * Set Data
+     * @param $key
+     * @param mixed|null $value
+     * @return $this
+     */
+    public function setData($key, $value = null)
+    {
+        if (is_array($key)) {
+            foreach ($key as $key1 => $value1) {
+                if (is_scalar($key1)) {
+                    $this->setData($key1, $value1);
+                }
+            }
+        } elseif (is_scalar($key)) {
+            $this->data[$key] = $value;
+        } else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method setData uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-woocommerce-checkout/pullRequest?prid=5510424)